### DISCUSSION TITLE
Troubleshooting (Traceback Error)

### DISCUSSION THREAD

**Discussion by cherryglazer** [Monika-After-Story/MonikaModDev#9416] [QA - tech support + general]
Hello!

I've been receiving this traceback error for quite a few days now, and the DDLC app will no longer even open for me. I've checked my files umpteen times and re-installed MAS once again (w/out deleting my old files), but nothing seems to work.

Here is what the error says:
![image](https://user-images.githubusercontent.com/110358748/182048641-0232a63c-b959-4d4a-aa09-c66f09565d98.png)
... and so on & so forth

"spaceroom" as a .png files also shows up in my files, contrary to the traceback:
![image](https://user-images.githubusercontent.com/110358748/182048731-190df22b-972b-472f-a895-381584eb0b58.png)

If anyone can help, I'd greatly appreciate it !!

**Comment by Booplicate** [comment]
Send your `traceback.txt`

**Comment by cherryglazer** [comment]
[traceback.txt](https://github.com/Monika-After-Story/MonikaModDev/files/9229903/traceback.txt)

**Comment by Booplicate** [comment]
> `lib/windows-i686/script-story-events.rpyc`

You installed the mod into the wrong folder. Make a fresh DDLC install and extract MAS into `game/`.

**Reply by cherryglazer** [reply]
> Hmm, it still doesn't appear to be working.

**Reply by Booplicate** [reply]
What's the error now? What did you do?
